sigle/src/ui/IconButton.ts

```diff
@@ -6,8 +6,16 @@ export const IconButton = styled('button', {
   alignItems: 'center',
   fontSize: '$2',
 
+  '& svg': {
+    color: '$gray11',
+  },
+
   variants: {
     size: {
+      sm: {
```

Comment:
sm seems to have the same values as md here 🤔

Reply:
Oh I think the value for sm was actually meant to be `$1` for padding, but after looking at the design system I can see there is only one size for the Icon Button with padding of `$1` so will remove the sm and add that change.